unidist/core/backends/mpi/core/communication.py

```diff
@@ -365,6 +382,12 @@ def send_complex_data(comm, data, dest_rank):
         A list of pickle buffers.
     list
         A list of buffers amount for each object.
+
+    Notes
+    -----
+    A blocking send is used when waiting for a completion that is necessary
+    for the pipeline to continue, or when the receiver is waiting for a result.
+    Otherwise use non-blocking function ``isend_complex_data``
```

Comment:
```suggestion
    This blocking send is used when we have to wait for completion of the communication,
    which is necessary for the pipeline to continue, or when the receiver is waiting for a result.
    Otherwise, use non-blocking ``isend_complex_data``.
```